### ISSUE TITLE
Error Signature is not valid.

### ISSUE THREAD

**Issue by leffw** [ln-markets/api-python#8]
I'm having problem using the function `swap()`:
```bash
{'code': 'apiKeyInvalidSignature', 'message': 'Signature is not valid'}
```

**Comment by RomainRouphael** [comment]
Hey, maybe you need to use a new API key with permissions set for Swap?

**Comment by leffw** [comment]
> Hey, maybe you need to use a new API key with permissions set for Swap?

Yes, the permissions are enabled I tried using other resources and it worked only for this swap that had a problem.

**Comment by leffw** [comment]
resolved
